src/build/_helpers.py

```diff
@@ -0,0 +1,117 @@
+import functools
```

Comment:
`_helpers` is the new `_utils`? :laughing:

Reply:
It seemed like the best place to put it 😛